bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcUniqueConfig.scala

```diff
@@ -19,7 +19,9 @@ import scalaz.Scalaz._
  * Unique configurations represent sets of observes in a sequence which are done with the same
  * instrument configuration and exposure times.
  */
-case class ItcUniqueConfig(count: Int, labels: String, config: Config) {
+case class ItcUniqueConfig(count: Int, labels: String, configs: Seq[Config]) {
+
+  def config = configs.head
```

Comment:
+1 for `NonEmptyList` ... also please avoid using`Seq` because it's impossible to reason about; you must assume that any `Seq` is both infinite and mutable, which drastically limits the safe operations you can perform.